modules/openfed_features/openfed_administration/openfed_administration.module

```diff
@@ -95,6 +95,9 @@ function openfed_administration_node_type_add_form_submit(array &$form, FormStat
   // Add content moderation to the new content type.
   \Drupal::service('openfed_administration.content_type_manager')->addWorkflowToBundle($bundle->id());
 
+  // Enable translation for the new content type.
```

Comment:
Why do we need this?
The language definition is being added a bit before than this code.